hacking/aws_config/test-policies.yml

```diff
@@ -0,0 +1,60 @@
+# Usage: ansible-playbook test-policies.yml -vv
+#
+- hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: 'env python'
+
+    aws_region: "{{ region | default('us-east-1') }}"
+    iam_group_prefix: "{{ group_prefix | default('ansible-integration') }}"
+    iam_policy_prefix: "{{ policy_prefix | default('AnsibleIntegratation') }}"
+    aws_account_id: "{{ test_account_id | default('123456789012') }}"
+    ci_policies: '../../aws/policy/*.yaml'
+    test_policies: 'test_policies/*.yaml'
+
+  tasks:
+    - name: 'Check size of policies'
+      with_fileglob:
+        - '{{ test_policies }}'
+        - '{{ ci_policies }}'
+      set_fact:
+        policy: "{{ lookup('template', item) | from_yaml | to_json }}"
+      changed_when:
```

Comment:
I was trying to ensure that all failures could be identified at once rather than run, fix, run, fix.  Assert and failed both result in an instant failure when just one fails.  When I tried ignore_errors it didn't try the other entries in the list.